03/app.js

```diff
@@ -41,13 +41,18 @@ books.getAuthor = function(isbn) {
     return false;
 }
 
-books.getTitle = function(isbn, lang) {
-
-}
-
-books.getTranslator = function(isbn, lang) {
-
+books.getTitle = function (isbn, lang) {
+    return this?.[isbn] && this[isbn] !== undefined ? this[isbn]['title'][lang]:null
 }
+books.getTranslator = function (isbn, lang) {
+    if (!this[isbn]['translator'][lang]) {
```

Comment:
Tutaj też powinieneś sprawdzić czy `this[isbn]` istnieje bo inaczej będzie błąd.